modules/ROOT/pages/modify-endpoints.adoc

```diff
@@ -0,0 +1,186 @@
+|<host> keystore | Yes
+|Maximum retry attempts | No
+|MDN signature algorithm | No
+|Message subject | No
+|<partner> AS2 identifier | No
+|<partner> AS2 URL | No
+|Request receipt | No
+|Respond to email address | No
+|Retry interval | No
+|Signature algorithm | No
+|===
+
+[[ftp-receive]]
+== FTP and SFTP Receive Endpoints
+
+You can modify the following fields for an FTP and SFTP Receive from Partners or Source at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Archival path |Yes
+|Endpoint name |No
+|Desription | No
+|File name pattern |Yes
+|Host |Yes
+|Password |Yes
+|Polling frequency |Yes
+|Port | Yes
+|Size check wait time |Yes
+|Source path |Yes
+|Username | Yes
+|===
+
+[[ftp-send]]
+== FTP and SFTP Send Endpoints
+
+You can modify the following fields for an FTP and SFTP Send to Partners or Target at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Description |No
+|Endpoint name |No
+|File name pattern |No
+|Host |No
+|Password |No
+|Port |No
+|Target path |No
+|Username |Yes
+|===
+
+[[http-receive]]
+== HTTP Receive Endpoints
+
+You can modify the following fields for an HTTP Receive from Partners or Source at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Connection idle timeout (in milleseconds) | Yes
+|Description | Yes
+|Endpoint name | Yes
+|Manage with API Manager and API Auto discovery id | Yes, if the *Private port (when deploying to VPC)* field is enabled
+|===
+
+[[http-send]]
+== HTTP Send Endpoints
+
+You can modify the following fields for an HTTP Send to Partners or Target at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Authentication type and associated fields a| 
+* No, for the Authentication type field
+* Yes, for fields associated with the authentication type
+|Endpoint name | No
+|Description | No
+|URL | No
+|Response timeout (in milliseconds) | No
+|===
+
+[[https-receive]]
+== HTTPS Receive Endpoints
+
+You can modify the following fields for an HTTPS Receive from Partners or Source at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Authentication type and associated fields | Yes
+|Connection idle timeout (in milleseconds) | Yes
+|Credentials | Yes
+|Description | Yes
+|Endpoint name | Yes
+|Manage with API Manager and API Auto discovery id | Yes, if the *Private port (when deploying to VPC)* field is enabled
+|===
+
+[[https-send]]
+== HTTPS Send Endpoints
+
+You can modify the following fields for an HTTPS Send to Partners or Target at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Authentication type and associated fields
+ a| 
+ * No, for the Authentication type field
+ * Yes for fields associated with the authentication type
```

Comment:
It should be just "Yes" - with no bullets. 
When Authentication type is changed, associated fields also change.